app/javascripts/helpers/web3Service.jsx

```diff
@@ -169,7 +170,7 @@ class Web3Service {
   }
 
   makeEthFromWei(weiValue) {
-    return parseFloat(this.web3.fromWei(weiValue, 'ether').valueOf(), 10);
+    return Big(this.web3.fromWei(weiValue, 'ether').valueOf());
```

Comment:
After I try over a method in Web3.js following your advice, I found it.
https://github.com/ethereum/wiki/wiki/JavaScript-API#web3tobignumber
Thxs. I will replace it with that.